spec/lib/facebook_data_analyzer/analyzeables/friends_spec.rb

```diff
@@ -0,0 +1,38 @@
+RSpec.describe FacebookDataAnalyzer::Friends do
+  subject do
+    described_class.new(catalog: test_catalog)
+  end
+
+  before(:each) do
+    subject.analyze
+  end
+
+  describe '#analyze' do
+    it 'should return proper friends details' do
+      friend = subject.friends.first
```

Comment:
What I find a little bit nicer is, when you extract `friend` into a `let` statement.

```
  describe '#analyze' do
    let(:friend) { subject.friends.first }

    it 'should return proper friends details' do
      expect(friend.name).to eq('Isaac Perez')
      expect(friend.date_added.to_s).to eq("2017-11-05T00:00:00+00:00")
```

Reply:
@thnukid: 👍 I will extract friend, contact and message in spec files into lets.